transition-scripts/helpers/helm.sh

```diff
@@ -127,8 +127,16 @@ get_vanity_route_name() {
   if [ "$#" -lt 1 ]; then exit 1; fi
 
   namespace="$1"
-
-  if [ "$namespace" = "c6af30-dev" ]
+  if [ "$namespace" = "e4ca1d-dev" ]
+  then
+    KEYCLOAK_ROUTE="sso-dev-sandbox"
+  elif [ "$namespace" = "e4ca1d-test" ]
+  then
+    KEYCLOAK_ROUTE="sso-test-sandbox"
+  elif [ "$namespace" = "e4ca1d-prod" ]
+  then
+    KEYCLOAK_ROUTE="sso-prod-sandbox"
+  elif [ "$namespace" = "c6af30-dev" ]
   then
     KEYCLOAK_ROUTE="sso-dev-sandbox"
   elif [ "$namespace" = "c6af30-test" ]
```

Comment:
@thegentlemanphysicist do you still need this config as we no longer intend to use c6??

Reply:
There's a bunch of references to the old-sandbox left in the code, I was going to leave them until we completed the migration entirely, (incase we needed to undo it )

Reply:
but I can take it out pretty easily